### DISCUSSION TITLE
how to uninstall

### DISCUSSION THREAD

**Discussion by charlesmilam** [gopasspw/gopass#2249] [Q&A]
Simple question, how to uninstall?

**Comment by AnomalRoil** [comment]
On what operating system?
How did you install it?

**Comment by pythoninthegrass** [comment]
FWIW I interrupted my first run of gopass (i.e., `gopass setup`) and was trying to figure out where it was getting the password store from. Turns out it was in `~/.password-store` on macOS (maybe the same on Linux?). 

Can confirm by checking with `gopass config` which will show: `mounts.path = /Users/<username>/.password-store`

So to uninstall on the former:
```bash
rm -rf ~/.password-store
brew uninstall gopass
```
